config/flownode.example.toml

```diff
@@ -5,6 +5,10 @@ mode = "distributed"
 ## @toml2docs:none-default
 node_id = 14
 
+## The number of flow worker in flownode.
+## Not setting this value will use the number of CPU cores divided by 2.
+#+num_workers=8
```

Comment:
Sugg: also provides `0` as default value (which is num cores divided by 2)

Reply:
> Sugg: also provides `0` as default value (which is num cores divided by 2)

a little bit wired that config has to be `mut` all the way down and everywhere that use it need to check for non zero, maybe just validate them to be greater than zero is good enough?or make num_workers a `NonZero<usize>` instead?